app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -166,32 +167,17 @@ private fun HelpCenterHomeScreen(
           )
         }
         Spacer(Modifier.height(40.dp))
-        AnimatedVisibility(
-          visible = quickLinks.isNotEmpty(),
-          enter = fadeIn() + expandVertically(clip = false, expandFrom = Alignment.CenterVertically),
-          exit = fadeOut() + shrinkVertically(clip = false, shrinkTowards = Alignment.CenterVertically),
-        ) {
-          HelpCenterSection(
-            title = stringResource(R.string.HC_QUICK_ACTIONS_TITLE),
-            chipContainerColor = MaterialTheme.colorScheme.typeContainer,
-            contentColor = MaterialTheme.colorScheme.onTypeContainer,
-            content = {
-              Column(
-                verticalArrangement = Arrangement.spacedBy(4.dp),
-              ) {
-                for (quickLink in quickLinks) {
+        HelpCenterSection(
+          title = stringResource(R.string.HC_QUICK_ACTIONS_TITLE),
+          chipContainerColor = MaterialTheme.colorScheme.typeContainer,
+          contentColor = MaterialTheme.colorScheme.onTypeContainer,
+          content = {
+            Column(
+              verticalArrangement = Arrangement.spacedBy(4.dp),
+            ) {
+              if (isLoadingQuickLinks) {
+                List(5) {
```

Comment:
Also do you think we can extract everything from this `List(5) {` to the end of it to a private `QuickActionPlaceholders` composable just to keep things a bit easier to read here